examples/KitchenSink/src/LINEBot/KitchenSink/EventHandler/MessageHandler/AudioMessageHandler.php

```diff
@@ -52,6 +52,21 @@ public function __construct($bot, $logger, \Slim\Http\Request $req, AudioMessage
 
     public function handle()
     {
+        $replyToken = $this->audioMessage->getReplyToken();
+
+        $contentProvider = $this->audioMessage->getContentProvider();
+        if ($contentProvider->isExternal()) {
+            $resp = $this->bot->replyMessage(
+                $replyToken,
+                new AudioMessageBuilder(
+                    $contentProvider->getOriginalContentUrl(),
+                    $this->audioMessage->getDuration()
+                )
+            );
+            $this->logger->info($resp->getRawBody());
```

Comment:
I do not know why, but only because AudioMessageHandler "info log" was called in following.
https://github.com/line/line-bot-sdk-php/pull/216/files#diff-387ac25c1588e729fa06871ffa1588c3L75

Reply:
Removed.